### PR TITLE
Use of checkers on docker_cli_import_test.go

### PR DESCRIPTION
**Pull request by liaoqingwei**
Signed-off-by: liaoqingwei liaoqingwei@huawei.com

### CODE REVIEW

File: integration-cli/docker_cli_import_test.go [hunk 1]
```diff
@@ -20,30 +21,20 @@ func (s *DockerSuite) TestImportDisplay(c *check.C) {
...
-	if err != nil {
-		c.Errorf("import failed with errors: %v, output: %q", err, out)
-	}
+	c.Assert(err, checker.IsNil)
 
-	if n := strings.Count(out, "\n"); n != 1 {
-		c.Fatalf("display is messed up: %d '\\n' instead of 1:\n%s", n, out)
-	}
-	image := strings.TrimSpace(out)
+	c.Assert(strings.Count(out, "\n"), checker.Equals, 1, check.Commentf("display is expected 1 '\\n' but didn't"))
```
Comment (vdemeester) [inline integration-cli/docker_cli_import_test.go]: There is now a `Count` checker :wink: 
`c.Assert(out, checker.Count, "\n", 1, check.Commentf(…))`

File: integration-cli/docker_cli_import_test.go [hunk 2]
```diff
@@ -51,29 +42,21 @@ func (s *DockerSuite) TestImportFile(c *check.C) {
...
-	if err != nil {
-		c.Fatal("failed to export a container", err)
-	}
+	c.Assert(err, checker.IsNil, check.Commentf("failed to export a container"))
 
 	out, _ := dockerCmd(c, "import", temporaryFile.Name())
-	if n := strings.Count(out, "\n"); n != 1 {
-		c.Fatalf("display is messed up: %d '\\n' instead of 1:\n%s", n, out)
-	}
+	c.Assert(strings.Count(out, "\n"), checker.Equals, 1, check.Commentf("display is expected 1 '\\n' but didn't"))
```
Comment (vdemeester) [inline integration-cli/docker_cli_import_test.go]: Same here :stuck_out_tongue:

File: integration-cli/docker_cli_import_test.go [hunk 3]
```diff
@@ -81,48 +64,34 @@ func (s *DockerSuite) TestImportFileWithMessage(c *check.C) {
...
-		c.Fatal("failed to export a container", err)
-	}
+	c.Assert(err, checker.IsNil, check.Commentf("failed to export a container"))
 
 	message := "Testing commit message"
 	out, _ := dockerCmd(c, "import", "-m", message, temporaryFile.Name())
-	if n := strings.Count(out, "\n"); n != 1 {
-		c.Fatalf("display is messed up: %d '\\n' instead of 1:\n%s", n, out)
-	}
+	c.Assert(strings.Count(out, "\n"), checker.Equals, 1, check.Commentf("display is expected 1 '\\n' but didn't"))
```
Comment (vdemeester) [inline integration-cli/docker_cli_import_test.go]: And here :wink:

File: integration-cli/docker_cli_import_test.go (hunk 1 above)
Comment (liaoqingwei) [inline integration-cli/docker_cli_import_test.go]: Hah,checker is so nice...,I will update it.

File: integration-cli/docker_cli_import_test.go (hunk 3 above)
Comment (liaoqingwei) [inline integration-cli/docker_cli_import_test.go]: Thanks @vdemeester, please review it again。

**Comment by vdemeester** [comment]
LGTM 🐻

**Comment by HuKeping** [comment]
seems good to go ping @cpuguy83

**Comment by calavera** [comment]
LGTM

**Pull request merged**